WordPress/Docs/WP/DiscouragedConstantsStandard.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0"?>
+<documentation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://phpcsstandards.github.io/PHPCSDevTools/phpcsdocs.xsd"
+    title="Discouraged Constants"
+    >
+    <standard>
+    <![CDATA[
+    The usage of WordPress Constants is discouraged.
```

Comment:
I wonder if it makes sense to list here and in the other `<standard>` block the discouraged constants, as this sniff triggers a warning only for a specific set of WP constants and not all constants. What do you think?